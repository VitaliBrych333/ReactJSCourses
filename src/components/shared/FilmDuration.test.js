import React from 'react';
import Duration from './FilmDuration';
import renderer from 'react-test-renderer';

describe('<Duration/>', () => {
  it('should equals to snapshot of Duration', () => {
    const mockPropValue = {
      release_date: '2019-12-12',
      runtime: '12'
    };
    const renderedValue = renderer.create(<Duration propValue={mockPropValue}/>).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
})
