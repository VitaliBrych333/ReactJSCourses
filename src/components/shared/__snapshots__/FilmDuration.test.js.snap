// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Duration/> should equals to snapshot of Duration 1`] = `
<div
  className="sc-bdVaJa dSanoj"
>
  <p>
    <span>
      2019
    </span>
     year
  </p>
  <p>
    <span>
      12
    </span>
     min
  </p>
</div>
`;
